_assets/js/templates/badges/badgeLogic.js

```diff
@@ -0,0 +1,24 @@
+import { replaceState } from '../../utils/replaceHistory';
+import toggleSelectorState from '../../utils/taSelectors/updateTASelectors';
+import toggleVisibility from '../../utils/taSelectors/toggleVisibility';
+import totalResults from '../search/totalResultsTemplate';
+
+export default function badgeLogic(element, state) {
+  // Get our unordered list to put badges in:
+  const badgeList = document.querySelector('#selector-badges ul');
```

Comment:
Only if the script is being imported on a page it shouldn't be. The template that creates the #selector-badges ul and the template that adds the bundled JS both check that the page title is 'Resources'. This could only be an issue if the JS starts being used on another page.